bin/BackupPC

```diff
@@ -489,7 +490,7 @@ sub Main_TryToRun_nightly
             #
             delete($Info->{poolKb2});
             delete($Info->{cpoolKb2});
-            for my $p ( qw(pool cpool) ) {
+            for my $p (qw(pool cpool)) {
```

Comment:
@shancock9 Regarding the new feature request https://github.com/perltidy/perltidy/issues/26

Just found this new bug, this could be another edge case similar to foreach.